app/helpers/generic_object_helper/textual_summary.rb

```diff
@@ -26,6 +26,22 @@ def textual_group_attribute_details_list
     )
   end
 
+  def textual_group_associations
+    associations = %i()
```

Comment:
this could be something like
`associations =  @record.property_associations.collect ...`

but it can be done in a follow-up PR